### ISSUE TITLE
Base typography styles are added to wp-core-ui items

### ISSUE THREAD

**Issue by ronilaukkarinen** [digitoimistodude/air-light#118]
Selectors for typographic elements are too wide:

https://github.com/digitoimistodude/air-light/blob/843bf347626e10d6ff62cc5ecc4662a95178d290/sass/layout/_typography.scss#L25-L48

This causes things like this:

![image](https://user-images.githubusercontent.com/1534150/121494870-cd0df280-c9e1-11eb-9555-5eff01b30b62.png)

I've been testing out with this:

``` css
body:not(.wp-core-ui),
.editor-styles-wrapper *:not(.wp-core-ui) {
```

But in here we have way too specific definitions that may cause problems in the front-end. All wp-admin core UI elements have been marked by `wp-core-ui` class so we could use that. If we disable the wrapper alltogether we lose for example the default paragraph styles inside blocks because wp-core-ui styles will then override them (too little base fonts for example).

I'm testing this out in the next projects and when the best possible fix is found will implement it to Air. I'm all open up to better suggestions.

**Comment by ronilaukkarinen** [comment]
The previous suggestion causes problems on front-end.

This seems to work better in gutenberg-editor-styles.scss:

``` scss
.editor-styles-wrapper *:not(.wp-core-ui) {
  @import 'layout/typography';
}
```

Let's see how it goes.

**Comment by ronilaukkarinen** [comment]
This seems to be fixed in changes after PR #123 so closing the issue.
